Makefile

```diff
@@ -130,14 +130,14 @@ bench: fmt
 test:
 	go test -short -tags='debug testing netgo' -timeout=5s $(pkgs) -run=. -count=$(count)
 
-test-long: lint lint-ci
+test-long: lint lint-ci start-mongo
 	@mkdir -p cover
 	GORACE='$(racevars)' go test -race --coverprofile='./cover/cover.out' -v -failfast -tags='testing debug netgo' -timeout=60s $(pkgs) -run=$(run) -count=$(count)
-
-test-int: test-long start-mongo
 	GORACE='$(racevars)' go test -race --coverprofile='./cover/cover.out' -v -tags='testing debug netgo' -timeout=600s $(integration-pkgs) -run=$(run) -count=$(count)
 	-make stop-mongo
 
+test-int: test-long
```

Comment:
If we do keep this, please leave a comment explaining the difference between `test-long` and `test-int`.

Reply:
A short comment to that effect would satisfy me:

```
# Having this in place keeps the "testing interface" of the service compatible with accounts, blocker and so on.
```

Reply:
Personally I would remove `test-int` in favor for more explicit options.

`test-long` is a standard, that is find. 

for tests that require mongo, we could either make a `test-long-mongo` or devs know which tests need to use the `start-mongo` command before running. 

or `test-long` allows runs mongo so it is never a concern. 

because if one of the goals with `test-int` is so you only need to run the integration services for the test that need the integration services, then that will be invalid if we add a second service, unless all those tests require both mongo and this new service. 

tl;dr, I think we are splitting hairs. 

From my perspective, I would rather the tests take a few seconds longer and have `test-long` always run mongo so that I never have to deal with determining which test command I should use for which tests.